packages/loader/container-loader/src/container.ts

```diff
@@ -1626,16 +1624,18 @@ export class Container
 				// Start prefetch, but not set opsBeforeReturnP - boot is not blocked by it!
 				// eslint-disable-next-line @typescript-eslint/no-floating-promises
 				this.attachDeltaManagerOpHandler(
-					dmAttributes,
+					attributes,
 					loadMode.deltaConnection !== "none" ? "all" : "none",
+					lastProcessedSequenceNumber,
```

Comment:
I'm not 100% sure this is correct. I.e. why do we advance DeltaManager.lastProcessedSequenceNumber ?
I think it should only advance (over baseline snapshot sequence number) as result of playing ops.

Reply:
We were advancing it already in the first place for offline by doing this:
```typescript
// If we saved ops, we will replay them and don't need DeltaManager to fetch them
const sequenceNumber = pendingLocalState?.savedOps[pendingLocalState.savedOps.length - 1]?.sequenceNumber;
const dmAttributes = sequenceNumber !== undefined ? { ...attributes, sequenceNumber } : attributes;
```

It's a question I have as well, as I don't exactly know the answer to this. @anthony-murphy may be able to answer this question.

My understanding from testing and using the code tells me that by pushing these numbers forward, the deltaManager doesn't fetch those ops from the opstream.